content/en/security/code_security/iast/security_controls/_index.md

```diff
@@ -299,6 +299,62 @@ The following security control definition affects every `sql-sanitizer` package
 `SANITIZER:SQL_INJECTION:node_modules/sql-sanitizer/index.js:sanitize`
 
 
+{{% /collapse-content %}}
+
+{{% collapse-content title=".NET" level="h4" %}}
+
+## General syntax
+`KIND:VULNERABILITIES:Assembly:Class:Method(ParameterTypes)[:ParameterIndexes]`
```

Comment:
I would follow the same formatting and wording as the [format definition above](https://github.com/DataDog/documentation/blob/5a361c8667a4932d9822bfc52e8de0bca6b04308/content/en/security/code_security/iast/security_controls/_index.md?plain=1#L26).

KIND => TYPE
VULNERABILITIES => SECURE_MARKS

Reply:
Fine enough. Thankyou